scripts/run_validator_node.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+set -ue
+set -o pipefail
+
+sudo_cmd=""
+files_path=""
+chain_spec_name=""
+keychain_exists=""
+session_keys=""
+binary_path=""
+
+echo "This script will setup a Atleta Validator on your PC. Press Ctrl-C at any time to cancel."
+echo "Checking privileges... "
+if [ $(id -u) -eq 0 ]; then
+    echo "OK (root)"
+elif command -v sudo &> /dev/null; then
+    echo "OK (sudo)"
+    sudo_cmd=sudo
+else
+    echo "FAIL"
+    echo "Not running as root and no sudo detected. Please run this as root or configure sudo. Exiting." >&2
+    exit 1
+fi
+
+echo "Detecting your architecture... "
```

Comment:
Probably detecting the system arch is more important step than checking privileges. Also it can be useful to use arch variable to detect some features like sudo or root path.